qe_tools/parsers/_input_base.py

```diff
@@ -147,6 +151,11 @@ def __init__(self, content, *, qe_version=None):
             Valid version strings are e.g. '6.5', '6.4.1', '6.4rc2'.
         :type qe_version: Optional[str]
 
+        :param validate_species_names: A boolean flag (default: True) to enable
+            the consistency check between atom names and species names inferred
+            from the pseudopotential file name.
+        :type validate_species_names: Optional[bool]
```

Comment:
The type should be `bool`, without the `Optional[]` qualifier (which indicates that `None` can also be passed).

Reply:
right, done